src/routes/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+import { withRouteSpec } from "lib/middleware/index.ts"
+import { renderJsxToResponse } from "lib/render-jsx-to-response.ts"
+
+export default withRouteSpec({
+  auth: "none",
+  methods: ["GET"],
+  middlewares: [],
+})(async (_req, _ctx) => {
+  return await renderJsxToResponse(<h1>Hello world!</h1>)
```

Comment:
WTB middleware for
```suggestion
  return await ctx.jsx(<h1>Hello world!</h1>)
```

Reply:
can make an issue on edgespec for it, happy to add as long as it doesn't require all consumers to install & bundle React

Reply:
Could be the first edgespec plugin. I think `@edgespec/jsx` or  `@edgespec/react` would be need